### ISSUE TITLE
Suppress gnuplot kernel outputs after each cell

### ISSUE THREAD

**Issue by Subratappt** [has2k1/gnuplot_kernel#20]
## How to suppress the outputs after each cell 

![Screen Shot 2020-05-03 at 3 07 54 AM](https://user-images.githubusercontent.com/40207448/80892900-5afdb100-8ceb-11ea-91f8-92297f572957.png)

```
OS: Centos 7
gnuplot 4.6 patchlevel 2
Python 3.7.6

jupyter core     : 4.6.1
jupyter-notebook : 6.0.3
qtconsole        : 4.6.0
ipython          : 7.12.0
ipykernel        : 5.1.4
jupyter client   : 5.3.4
jupyter lab      : 1.2.6
nbconvert        : 5.6.1
ipywidgets       : 7.5.1
nbformat         : 5.0.4
traitlets        : 4.3.3
```

**Comment by has2k1** [comment]
This is a duplicate of #11. The problem with it is that it shows up on random installations.

**Comment by has2k1** [comment]
This has been fixed in the latest release v0.4.0.
